<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="panelMain">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbMachine1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>301</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Mezcladora 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbCarro">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>371</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Carro</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbMachine3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>410</y>
       <width>421</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Chocolatera</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbMachine2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>280</y>
       <width>341</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Mezcladora 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnIniciar">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>350</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>390</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbCola">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>311</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbRegistroMezcla0">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>500</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Total Mezcla: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lbRegistroMezcla">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>500</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbRegistroChoco0">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>500</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Total Chocolate: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lbRegistroChoco">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>500</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
    <widget class="QLabel" name="lbDatosMachine2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>340</y>
       <width>311</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>datos</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbDatosMachine1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>311</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>datos</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbDatosMachine3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>470</y>
       <width>311</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>datos</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
